Command/GenerateDoctrineCrudCommand.php

```diff
@@ -185,10 +185,10 @@ protected function interact(InputInterface $input, OutputInterface $output)
         $format = $input->getOption('format');
         $output->writeln(array(
             '',
-            'Determine the format to use for the generated CRUD.',
+            'Determine the routing format to use for the generated CRUD.',
```

Comment:
I would revert those routing addition in the text (same below) and in the option description. Sure, it's only about the routing today, but it might be about something else tomorrow.